src/providers/ad/ad_gpo.c

```diff
@@ -4745,6 +4745,13 @@ static void gpo_cse_done(struct tevent_req *subreq)
         DEBUG(SSSDBG_CRIT_FAILURE,
               "ad_gpo_parse_gpo_child_response failed: [%d][%s]\n",
               ret, sss_strerror(ret));
+
+        if (ret == EINVAL) {
```

Comment:
I am not sure if this is the right way to go. In result we will have two separated `SSSDBG_CRIT_FAILURE` messages. With the latest SSSD log changes message like this can trigger backtrace dump. I think better idea would be to figure out how to merge those two messages together. Or just change `sss_strerror(ret)` result when `ret == EINVAL` to something more meaningful.